src/points/slice.ts

```diff
@@ -42,6 +42,7 @@ interface State {
   trackOnceActivities: {
     [key in PointsActivityId]?: boolean
   }
+  introHasBeenDismissed: boolean
```

Comment:
```suggestion
  hasSeenIntro: boolean
```
maybe?

Reply:
this is similar to my initial attempt! 😅

but Joe fairly noted that this flag will only turn on when the user deliberately presses the dismiss button. for instance, they can _see_ the intro and navigate back, or terminate the app. see this [comment](https://github.com/valora-inc/wallet/pull/5406#discussion_r1599312617)

Reply:
oh i actually wasn't expecting that behaviour...why would we want to show this screen again if the user navigates back without pressing the button? the user needs to do _something_ to exit the screen, which is an acknowledgement that they've seen the screen (there's no guarantee that a user will actually read + comprehend the messaging on the screen, we tap the big buttons to quickly dismiss things all the time...) i thought the straightforward thing to do is to only show this screen once and mark it as seen once the user navigates away...